### PR TITLE
Replace `get` with `readonly` in interfaces in Nightwatch.

### PR DESCRIPTION
**Pull request by garg3133**
For compatibility with Typescript versions older than 4.3.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## nightwatch ([<kbd>unpkg</kbd>](https://unpkg.com/browse/nightwatch@latest/))
was missing the following properties:
1. createClient
1. client
1. cli
1. runTests
1. CliRunner

<details>
<summary>as well as these 2 other properties...</summary>
<p>initClient, runner</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c179fc5fb2fc44e44b4633ef260fb3c67024a471
</p>

File: types/nightwatch/index.d.ts [hunk 1]
```diff
@@ -1831,9 +1831,9 @@ export type NightwatchPage = {
 };
 
 export interface NightwatchApiCommands {
-    get WEBDRIVER_ELEMENT_ID(): string;
-    get browserName(): string;
-    get platformName(): string;
+    readonly WEBDRIVER_ELEMENT_ID: string;
+    readonly browserName: string;
+    readonly platformName: string;
     __isBrowserName(browser: string, alternateName: string): boolean;
```
Comment (vaibhavsingh97) [inline types/nightwatch/index.d.ts:1837]: can you please add tests for these

**Review by vaibhavsingh97 — APPROVED** [review]
LGTM

**Comment by vaibhavsingh97** [comment]
Ready to merge

**Pull request merged**